ecs-agent/netlib/model/ecscni/appmesh_config.go

```diff
@@ -0,0 +1,79 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package ecscni
+
+import (
+	"fmt"
+
+	"github.com/aws/amazon-ecs-agent/ecs-agent/netlib/model/appmesh"
+)
+
+// AppMeshConfig contains the information needed to incoke the appmesh CNI plugin.
```

Comment:
nit: is there a typo here with `incoke`?

Reply:
should be `invoke` I think..

Reply:
Since there will be a follow up PR, I'm happy for this to be done in that one.